srepanel/common/types.go

```diff
@@ -1,8 +1,9 @@
 package common
 
 type Identity struct {
-	ID       uint64 `json:"id"`
-	Username string `json:"username"`
+	ID         uint64 `json:"id"`
+	Username   string `json:"username"`
+	AvatarHash string `json:"avatar"`
```

Comment:
Looks fine for now. I suggest putting user information in the SQLite database long term, so the JWT doesn't grow without bounds.